nuit/templatetags/nuit.py

```diff
@@ -350,7 +352,10 @@ def render(self, context):
                     row_data.append((field_data, {}))
                 else:
                     try:
-                        row_data.append((field_name, literal_eval(data)))
+                        aeval = asteval.Interpreter()
```

Comment:
That throws an exception, as it tries to treat the `context` object as a real `dict`, which it isn't:

```python
Traceback (most recent call last):
  File "/Users/ben/lgit/hub/forks/django-nuit/nuit/tests/test_main.py", line 357, in test_specifying_a_width
    ''', {'form': self.form})
  File "/Users/ben/lgit/hub/forks/django-nuit/nuit/tests/test_main.py", line 301, in get_soup
    return soup(Template(template_contents).render(Context(context_dict)))
  File "/Users/ben/.virtualenvs/nuit/lib/python2.7/site-packages/django/template/base.py", line 206, in render
    return self._render(context)
  File "/Users/ben/.virtualenvs/nuit/lib/python2.7/site-packages/django/test/utils.py", line 92, in instrumented_test_render
    return self.nodelist.render(context)
  File "/Users/ben/.virtualenvs/nuit/lib/python2.7/site-packages/django/template/base.py", line 992, in render
    bit = node.render_annotated(context)
  File "/Users/ben/.virtualenvs/nuit/lib/python2.7/site-packages/django/template/base.py", line 959, in render_annotated
    return self.render(context)
  File "/Users/ben/lgit/hub/forks/django-nuit/nuit/templatetags/nuit.py", line 355, in render
    row_data.append((field_name, asteval.Interpreter(symtable=context)(data, show_errors=False)))
  File "/Users/ben/.virtualenvs/nuit/lib/python2.7/site-packages/asteval/asteval.py", line 129, in __init__
    for key, val in symtable.items():
AttributeError: 'Context' object has no attribute 'items'
```